src/Connections/SetupConnection.ts

```diff
@@ -216,7 +216,12 @@ export class SetupConnection extends CommandConnection {
         this.pushConnections(c.connection);
         const url = new URL(c.connection.hookId, this.config.generic.parsedUrlPrefix);
         const adminRoom = await this.getOrCreateAdminRoom(userId);
-        await adminRoom.sendNotice(`You have bridged a webhook. Please configure your webhook source to use ${url}.`);
+        await adminRoom.sendNotice(
+            `You have bridged the webhook "${name}" in https://matrix.to/#/${this.roomId} .\n` +
+                // Line break before and no full stop after URL is intentional.
+                // This makes copying and pasting the URL much easier.
```

Comment:
You may have these 8 spaces back to put them somewhere else. :yum: 
```suggestion
            // Line break before and no full stop after URL is intentional.
            // This makes copying and pasting the URL much easier.
```

Reply:
Since we started using Prettier in other repos, I have become really lazy :upside_down_face: